script/tool/test/update_excerpts_command_test.dart

```diff
@@ -33,12 +33,13 @@ void runAllTests(MockPlatform platform) {
       ));
   });
 
-  Future<void> testInjection(String before, String source, String after,
+  Future<void> testInjection(
+      String before, String source, String after, String fileName,
```

Comment:
This is a lot of positional arguments with the same type; ideally they should all be named, but lets at least name the new one.